config/crd/bases/pipelines.kubeflow.org_pipelineversions.yaml

```diff
@@ -49,6 +49,10 @@ spec:
                 type: string
               pipelineSpec:
                 x-kubernetes-preserve-unknown-fields: true
+              pipelineSpecURI:
```

Comment:
I'm confused because this was already added in dda21a38040816af9dea266fa8f094893af5526e.

Reply:
i too am also confused as your commit exists is somehow also present in this review (commit f4ef916d689723b7360d120259b2a23add6a4d2b)

Reply:
fixed by rebasing again, somehow